### PR TITLE
Blogpost for VS Code Language Support 0.8.0 release

### PR DESCRIPTION
**Pull request by djelinek**
cc @apupier

### CODE REVIEW

File: content/blog/2023/06/language-support-0.8.0-release/index.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+- Command to create a Camel route with Java DSL
+![Provide command to create a Camel route defined in Java DSL using Camel JBang](./camelRouteJavaDSL.gif)
+
+- Command to create a Camel route with XML DSL
+![Provide command to create a Camel route defined in XML DSL using Camel JBang](./camelRouteXMLDSL.gif)
+
+## Settings
+
+Beginning this release, there is new setting available for Apache Camel JBang version, which is used by extension commands:
```
Comment (apupier) [inline content/blog/2023/06/language-support-0.8.0-release/index.md:35]:
```suggestion
Beginning this release, there is a new setting available for Apache Camel JBang version, which is used by extension commands:
```

File: content/blog/2023/06/language-support-0.8.0-release/index.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+![Provide command to create a Camel route defined in Java DSL using Camel JBang](./camelRouteJavaDSL.gif)
+
+- Command to create a Camel route with XML DSL
+![Provide command to create a Camel route defined in XML DSL using Camel JBang](./camelRouteXMLDSL.gif)
+
+## Settings
+
+Beginning this release, there is a new setting available for Apache Camel JBang version, which is used by extension commands:
+
+- `Run Camel Application with JBang`
```
Comment (djelinek) [inline content/blog/2023/06/language-support-0.8.0-release/index.md:37]: need tu update these bullets

File: content/blog/2023/06/language-support-0.8.0-release/index.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+---
+title: "New release of VS Code Language Support for Camel 0.8.0"
+date: 2023-06-05
```
Comment (apupier) [inline content/blog/2023/06/language-support-0.8.0-release/index.md:3]: Can you update the date as we are already 2 days later? :(

**Review by apupier — APPROVED** [review]

**Comment by apupier** [comment]
ping usual suspect @oscerd . Can you review it please?

**Review by oscerd — APPROVED** [review]

**Pull request merged**